// image-generation-for-discord/src/index.js
module.exports.alert = require('./data/alert')
module.exports.batslap = require('./data/batslap')
module.exports.burn = require('./data/burn')
module.exports.caution = require('./data/caution')
module.exports.chineserestaurantsign = require('./data/chinese-restaurant-sign')
module.exports.cursedspongebob = require('./data/cursedspongebob')
module.exports.danger = require('./data/danger')
module.exports.dexter = require('./data/dexter')
module.exports.dislike = require('./data/dislike')
module.exports.drake = require('./data/drake')
module.exports.gay = require('./data/gay')
module.exports.genierules = require('./data/genierules')
module.exports.thisguy = require('./data/thisguy')
module.exports.girlworthfightingfor = require('./data/girlworthfightingfor')
module.exports.gruplan = require('./data/gru-plan')
module.exports.heman = require('./data/heman')
module.exports.hollywoodstar = require('./data/hollywoodstar')
module.exports.couldread = require('./data/couldread')
module.exports.kannapaper = require('./data/kannapaper')
module.exports.lisa = require('./data/lisa')
module.exports.mycollectiongrows = require('./data/mycollectiongrows')
module.exports.panik = require('./data/panik')
module.exports.rip = require('./data/rip')
module.exports.roadsign = require('./data/roadsign')
module.exports.spidermanpointing = require('./data/spidermanpointing')
module.exports.spongebobtimecard = require('./data/spongebobtimecard')
module.exports.strongpassword = require('./data/strongpassword')
module.exports.thatsignwontstopme = require('./data/thatsignwontstopme')
module.exports.tuxedopooh = require('./data/tuxedopooh')
module.exports.twobuttons = require('./data/twobuttons')
module.exports.worthless = require('./data/worthless')
module.exports.communism = require('./data/communism')
module.exports.avatarfusion = require('./data/avatar-fusion')
module.exports.snapchat = require('./data/snapchat')
module.exports.facts = require('./data/facts')
module.exports.deepfry = require('./data/deepfry')
module.exports.itsbeenyears = require('./data/3000years')
module.exports.motionblur = require('./data/motionblur')
